src/site/content/en/blog/share-like-a-native/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Share like a native app
+subhead: Web apps can use the same system provided share capabilities as native apps.
+authors:
+  - joemedley
+date: 2019-11-08
+hero: hero.jpg
```

Comment:
Don't want to block, but given the similarities between Web Share and Web Share Target, would it be helpful to use a different hero image to help differentiate the two posts?